### PR TITLE
Add Latino, *><>, ><>, Charcoal, Candy, Alice, BBC BASIC, C--, Cubix, DB2, GOTO++, Haifuckqueue, Husk, Haystack, Integral, Lost, Pyth, REBMU, Rotor, Evil and Your Mom

### PR DESCRIPTION
**Pull request by jixperson**
Hi, I removed all of the changes and added the languages from scratch \
    on commit bases with commit message and PR title.

## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme
- [x] I have no association with the language
- [x] There are no copyright issues with this code
- [x] The language has not been added prior to this pull request

#### Link to programming language: 
[*><>](https://github.com/redstarcoder/go-starfish)
[><>](https://gist.github.com/anonymous/6392418)
[Charcoal](https://github.com/somebody1234/Charcoal)
[Candy](https://github.com/dale6john/candy)
[Alice](https://github.com/m-ender/alice)
[BBC BASIC](http://www.bbcbasic.co.uk/bbcbasic.html)
[C--](https://www.microsoft.com/en-us/research/wp-content/uploads/1998/01/pal-manual.pdf)
[Cubix](https://github.com/ETHproductions/cubix/)
[DB2](https://www.ibm.com/docs/zosbasics/com.ibm.db2z.doc.intro/)
[GOTO++](https://gotopp.org/index.en.html)
[Haifuckqueue](https://github.com/4gboframram/Haifuckqueue)
[Husk](https://github.com/barbuz/Husk)
[Haystack](https://github.com/kade-robertson/haystack/)
[Integral](https://github.com/nph278/integral)
[Lost](https://github.com/Wheatwizard/Lost)
[Pyth](https://github.com/isaacg1/pyth)
[REBMU](http://rebmu.hostilefork.com/)
[Rotor](https://github.com/quartata/rotor-lang)
[Evil](https://web.archive.org/web/20160428204543/http://pastebin.ca/2313538)
[Your Mom](https://github.com/ConorOBrien-Foxx/your-mom)

### CODE REVIEW

**Review by CyberPotat42 — APPROVED** [review]

File: a/Alice.alice [hunk 1]
```diff
@@ -0,0 +1 @@
+"&O@!dlroW ,olleH
```
Comment (MrBrain295) [inline a/Alice.alice:1]:
```suggestion
"&O@dlroW olleH
```
Does it need the `,` and `!`

File: e/Ed [hunk 2]
```diff
@@ -0,0 +1,25 @@
+The following tab indented lines will cause a "original Unix text editor" (Ed)
```
Comment (MrBrain295) [inline e/Ed:1]: The comments need removed.

File: f/Fish.fish [hunk 3]
```diff
@@ -1,2 +1,2 @@
-#!/usr/bin/env fish
-echo "Hello World"
+"hello, world"r\
```
Comment (MrBrain295) [inline f/Fish.fish:1]: Can you revert this change?

File: h/Haifuckqueue [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{H}$1
+{lo, W}
+p{l}p
+{e}$7
+{orld!}
```
Comment (MrBrain295) [inline h/Haifuckqueue:5]:
```suggestion
{orld}
```

File: i/Integral [hunk 5]
```diff
@@ -0,0 +1 @@
+⌡Hello, World!
```
Comment (MrBrain295) [inline i/Integral:1]:
```suggestion
⌡Hello World
```

File: p/Pyth.pyth [hunk 6]
```diff
@@ -0,0 +1 @@
+"Hello"
```
Comment (MrBrain295) [inline p/Pyth.pyth:1]: Does this print `Hello World`?

File: y/Your Mom [hunk 7]
```diff
@@ -0,0 +1 @@
+'Hello,World
```
Comment (MrBrain295) [inline y/Your Mom:1]: Does this print `Hello World`?

**Review by MrBrain295 — CHANGES_REQUESTED** [review]

File: f/Fish.fish (hunk 3 above)
Comment (thelastsoyce11) [reply f/Fish.fish:1]: This is not the fish shell, i changed the name to ><> and Starfish to *><>
And i will fix the , and ! on the next commits.

File: p/Pyth.pyth (hunk 6 above)
Comment (thelastsoyce11) [reply p/Pyth.pyth:1]: fixed.

File: h/Haifuckqueue (hunk 4 above)
Comment (thelastsoyce11) [reply h/Haifuckqueue:5]: fixed.

File: a/Alice.alice (hunk 1 above)
Comment (thelastsoyce11) [reply a/Alice.alice:1]: #Fixed
Should be like: https://tio.run/##S8zJTE79/18pJacoP1whPycn1UMpRc0/weH/fwA.

File: e/Ed (hunk 2 above)
Comment (Richienb) [reply e/Ed:1]: This file parallels https://github.com/leachim6/hello-world/blob/main/v/Vi. What to do?

File: w/Wierd.w [hunk 8]
```diff
@@ -0,0 +1,10 @@
+-*******************           !dlroW ,olleH
```
[the remark this answers is not in the document]
Comment (thelastsoyce11) [reply w/Wierd.w]: OK i will check one by one to see if and change the messages.

**Comment by MrBrain295** [comment]
@thelastsoyce11 could you separate this into 2/3 pull requests?

**Review by thelastsoyce11 — COMMENTED** [review]
OK, Thanks.

**Comment by thelastsoyce11** [comment]
> 

Which parts should i separate

**Comment by MrBrain295** [comment]
Just split the first half by alphabetical order into one pull request and the other half into another pull request.

**Comment by Richienb** [comment]
I don't think there's any point in doing that now since it would just waste time.

**Review by CyberPotat42 — APPROVED** [review]

File: g/GOTO++.gpp [hunk 9]
```diff
@@ -0,0 +1,7 @@
+GOTO Le premier "Hello World"
+
+§100
+GOTOPRINTDUTEXTE() ; *`s' *`r' *`e' *`v' *`i' *`n' *`u' *` ' *`o' *`l' *`l' *`e' *`h'
```
Comment (Richienb) [inline g/GOTO++.gpp:4]: ?

Comment (MrBrain295) [reply g/GOTO++.gpp:4]: This is the Hello World example from their examples: GOTO Le premier "Hello world"

§100
GOTOPRINTDUTEXTE() ; *`s' *`r' *`e' *`v' *`i' *`n' *`u' *` ' *`o' *`l' *`l' *`e' *`h'
§2989
GOTOPRINTDUTEXTE() ; *`d' *`l' *`r' *`o' *`w' *` ' *`o' *`l' *`l' *`e' *`h'
GOTOGOTO() %2989 ?50 %100 ?40

Comment (jixperson) [reply g/GOTO++.gpp]: fixed know, sorry for the late response i got a fever.

**Review by MrBrain295 — APPROVED** [review]

**Review by Richienb — APPROVED** [review]
